src/app/components/Article/index.jsx

```diff
@@ -5,17 +5,23 @@ import Header from '../Header';
 import MainContent from '../../containers/MainContent';
 import mainContentPropTypes from '../../models/propTypes/mainContent';
 
-const Article = ({ lang, title, blocks }) => (
-  <Fragment>
-    <Helmet htmlAttributes={{ lang }}>
-      <title>{title}</title>
-    </Helmet>
-    <Header />
-    <MainContent blocks={blocks} />
-  </Fragment>
-);
+const Article = ({ lang, title, blocks, id }) => {
+  const canonicalLink = `https://www.bbc.com/news/article/${id}`;
```

Comment:
We will soon move all metadata to it's own component, which we just inject. This is fine here for now. just an FYI.